src/Caliburn.Micro.Core/Screen.cs

```diff
@@ -103,20 +103,33 @@ async Task IActivate.ActivateAsync(CancellationToken cancellationToken)
 
             if (!IsInitialized)
             {
+                try
+                {
+                    Log.Info("Initializing {0}.", this);
 #pragma warning disable CS0618 // Type or member is obsolete
-                await OnInitializeAsync(cancellationToken);
+                    await OnInitializeAsync(cancellationToken);
 #pragma warning restore CS0618 // Type or member is obsolete
-                IsInitialized = initialized = true;
-                await OnInitializedAsync(cancellationToken);
+                    IsInitialized = initialized = true;
+                    await OnInitializedAsync(cancellationToken);
```

Comment:
This is not something that was done here in this PR, but why was the name changed from OnInitializeAsync to OnInitializedAsync? This method is called when initializing so the initializing happens in the OnInitializeAsync thus the name OnInitializeAsync makes more sense then OnInitializedAsync since OnInitializedAsync  implies that the initialization is already completed.